pwa/src/templates/componentDetail/ComponentsDetailTemplate.module.css

```diff
@@ -52,7 +52,7 @@
 
 .cardsContainer {
   display: grid;
-  grid-template-columns: 2fr 2fr 2fr 1fr;
+  grid-template-columns: 2fr 1fr 1fr 1fr;
```

Comment:
It looks all right, however, a couple of notes (or thoughts):

1. We could remove the Github column and add the Github as an tag in the header (would seem more consistent);
2. The licenses would be better represented within a table, right now a tag within a column seems like an odd placement;